airflow/jobs/scheduler_job.py

```diff
@@ -327,6 +327,17 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session =
                 self.log.warning("Tasks using non-existent pool '%s' will not be scheduled", pool)
                 continue
 
+            pool_total = pools[pool]["total"]
+            if task_instance.pool_slots > pool_total:
+                self.log.warning(
+                    "Not executing %s since requesting %s slots when total pool '%s' slots are %s.",
+                    task_instance,
+                    task_instance.pool_slots,
+                    pool,
+                    pool_total,
+                )
```

Comment:
```suggestion
                self.log.warning(
                    "Not executing %s. Requested pool slots (%s) are greater than total pool slots: '%s' for pool: %s.",
                    task_instance,
                    task_instance.pool_slots,
                    pool_total,
                    pool,
                )
```

Reply:
WDYT?

Reply:
applied in https://github.com/apache/airflow/pull/20178/commits/6a7a9657f6deb6c5c58f0f96ffb2143318f0877d